.github/PR_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Proposed change(s)
+
+Describe the changes made in this PR.
+
+### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)
+
+
+
+### Types of change(s)
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Code refactor
+- [ ] Breaking change
+- [ ] Documentation update
+- [ ] Other (please describe)
+
+### Checklist
+- [ ] I have added tests that prove my fix is effective or that my feature works
```

Comment:
I would almost rather just have a section of "How was this change tested?"

Reply:
I suggest that we merge this template and then refine it further as we start using it.